include/dxc/DXIL/DxilConstants.h

```diff
@@ -60,6 +60,22 @@ inline int CompareVersions(unsigned Major1, unsigned Minor1, unsigned Major2,
   return 0;
 }
 
+// Utility for updating major,minor to max of current and new.
+inline bool MaxOfShaderModels(unsigned &major, unsigned &minor,
```

Comment:
Maybe move this to DxilShaderModel.h?

Reply:
Maybe I should consider a rename instead.  It really isn't specific to DxilShaderModel, it's just trying to do an in-place max for a major/minor version pair.

How about `UpdateToMaxOfVersions`?